vizro-ai/examples/dashboard_ui/components.py

```diff
@@ -0,0 +1,216 @@
+"""Contains custom components used within a dashboard."""
+
+from typing import List, Literal
+
+import black
+import dash_bootstrap_components as dbc
+import vizro.models as vm
+from dash import dcc, html
+from pydantic import PrivateAttr
+from vizro.models import Action
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._models_utils import _log_call
+
+
+class UserPromptTextArea(vm.VizroBaseModel):
```

Comment:
Possible future change: we have `vizro.models._components.form._text_area.TextArea` that might do this already?